.github/workflows/preview-deploy.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           PR_NUMBER: ${{ needs.pull-request-data.outputs.number }}
         run: |
           aws s3 sync ./html "s3://elastic-docs-v3-website-preview/${GITHUB_REPOSITORY}/pull/${PR_NUMBER}" --delete
-          aws cloudfront create-invalidation --distribution-id EKT7LT5PM8RKS --paths "/${GITHUB_REPOSITORY}/pull/${PR_NUMBER}/*"
+          aws cloudfront create-invalidation --distribution-id EKT7LT5PM8RKS --paths "/${GITHUB_REPOSITORY}/pull/${PR_NUMBER}*"
```

Comment:
Apologies if you're still working on this PR. I think this is breaking links to other pages.

Example: Clicking on `Contribute to Elastic documentation` takes you to

https://docs-v3-preview.elastic.dev/elastic/docs-builder/pull/contribute/index.html
instead of
https://docs-v3-preview.elastic.dev/elastic/docs-builder/pull/436/contribute/index.html

Reply:
This was actually my attempt to fix it 

I'm not sure whats going on.. I found the same behaviour in other previews.

Reply:
you will notice the links are working just fine at https://docs-v3-preview.elastic.dev/elastic/docs-builder/pull/436/

(trailing slash)

I don't know what the cause for this is.

Reply:
I found the culprit.

If the URL ends without a trailing slash https://docs-v3-preview.elastic.dev/elastic/docs-builder/pull/436

and a link has a relative path e.g. `<a href="./configure/index.html">Configure content sets in V3</a>`

then, the link is resolved to https://docs-v3-preview.elastic.dev/elastic/docs-builder/pull/configure/index.html

Reply:
Aha!

Reply:
https://github.com/elastic/docs-builder/pull/442 is a potential fix